templates/common/infra/bicep/core/host/apim.bicep

```diff
@@ -0,0 +1,60 @@
+param name string
```

Comment:
I don't think apim is a 'host' since we don't push code to it.  We'll likely need a new folder in core.  Something like 'gateway'?

Reply:
Ok

Reply:
I will move the apim.bicep to the gateway folder

Reply:
Based on what you know about the product is "gateway" the best "category"/ folder name?

Reply:
Yes I think gateway is a good name